### DISCUSSION TITLE
Is there a way to change the label color in the NavigationDrawer?

### DISCUSSION THREAD

**Discussion by kgall89** [flet-dev/flet#3527] [Q&A]
### Question

Ive found out how to change the icon color when selected and not selected but the label is staying a black/gray color. I found a thread that mentioned placing label_content, but this did not work as this argument was not found.

### Code sample

```python
ft.NavigationDrawer(
        bgcolor=blue,
        indicator_color=gray,
        on_dismiss=handle_dismissal,
        on_change=handle_change,
        controls=[
            ft.Container(height=12),

            ft.NavigationDrawerDestination(
                icon_content=ft.Icon(ft.icons.FACT_CHECK_OUTLINED, color=gray),
                label="Check In",
                selected_icon_content=ft.Icon(ft.icons.FACT_CHECK, color=blue)
            ),
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by J-P-Toledo** [comment]
I'm going through the exact same situation here. Got any answers elsewhere?
